test/test_nn.py

```diff
@@ -7206,6 +7206,12 @@ def multimarginloss_weights_no_reduce_test():
         input_fn=lambda: _rand_tensor_non_equal(1, 3, 5, 6),
         desc='tuple',
     ),
+    dict(
```

Comment:
there should also be tests that check the modules return expected sizes at these dimensions containing `None`